website/Gemfile.lock

```diff
@@ -79,18 +79,18 @@ GEM
       celluloid (~> 0.16.0)
       rb-fsevent (>= 0.9.3)
       rb-inotify (>= 0.9)
-    middleman (3.3.13)
+    middleman (3.3.12)
```

Comment:
I'm not sure what's the question/issue ? :)

I have not been changing in the actual `Gemfile`, all these packages are resolved dependencies.